src/assets/stylesheets/components/main.sass

```
@@ -0,0 +1,196 @@
+.main
+  background-image: linear-gradient(to bottom, $color-full-transparent 66%, $color-white), url('../../assets/images/backgrounds/beach.jpg')
+  background-size: cover
+  color: $color-white
+  min-height: 100vh
+  padding: 140px $m-base $m-3xl $m-base
+
+  .tab-bar
+    border-bottom: solid $color-light-transparent 2px
+    font-size: $font-size-sm
+    margin-bottom: $m-lg
+    padding-bottom: $m-sm
+    position: relative
+
+    &__item
+      font-weight: $font-weight-bold
+      letter-spacing: $spacing-wide
+      padding-bottom: $m-sm
+      text-transform: uppercase
+
+      &.-active
+        border-bottom: solid $color-white 2px
+
+  .side-cards
+    display: none
+
+  &__header
+    font-size: $font-size-3xl
+
+  &__description
+    font-size: $font-size-lg
+    font-weight: $font-weight-bold
+
+  &__details-container
+    border-left: solid $color-white 3px
+    margin: $m-lg 0
+
+  &__details
+    margin: $m-md $m-base
+
+  .main-btn
+    align-items: center
+    background-color: $color-light-transparent
+    border: none
+    color: $color-white
+    display: flex
+    justify-content: space-between
+    padding: 0
+
+    &__content
+      font-weight: $font-weight-bold
+      padding: $m-sm $m-lg
+
+    &__icon
+      background-color: $color-white
+      color: $brand-primary
+      height: $m-base * 2
+      padding: $m-sm
+      width: $m-base * 2
+
+  .search
+    color: $color-black
+    display: flex
+    flex-wrap: wrap
+    font-size: $font-size-sm
+    margin-top: $m-3xl
+
+    &__icon-container
+      display: none
+
+    &__icon
+      display: inline-block
+      height: $font-size-xs
+      margin-left: $m-xs
+
+      &.-transparent
+        height: $m-lg
+        margin: auto
+        opacity: .4
+        width: $m-lg
+
+    &__label
+      display: block
+      margin-bottom: $m-sm
+
+    &__section
+      background-color: $color-white
+      color: $color-gray-darkest
+      flex-basis: 40%
+      flex-grow: 1
+      font-size: $font-size-xs
+      padding: $m-base $m-md
+
+    &__dropdown-expand
+      +btn-reset
+
+    &__dropdown
+      align-self: center
+      color: $color-black
+      display: flex
+      font-size: $font-size-base
+      font-weight: $font-weight-bold
+
+    &__button
+      background-color: $brand-primary
+      border: 0
+      color: $color-white
+      flex-basis: 20%
+      flex-grow: 1
+      font-size: $font-size-lg
+      font-weight: $font-weight-bold
+      margin-top: $m-sm
+      padding: $m-md
+
++media-breakpoint-up(xl)
+  .main
+    padding: $m-base * 7 $m-3xl $m-3xl $m-3xl
+
+    .side-cards
+      display: flex
+      justify-content: space-between
+      position: absolute
+      right: 0
+      top: 26px
```

Magic number warning :)
```suggestion
      top: 100%
```

---

Wait, what? I don't get how it could work, but it actually does. I'll ask you for explanation, once you're back in the office 😄